### DISCUSSION TITLE
file not opening

### DISCUSSION THREAD

**Discussion by OlaS2020** [umair-hassan/best-toolbox#33] [General]
I am new to using BEST, following your Brain initiative presentation, but the mat data file is not opening? any advice. 
Also does BEST only work on mt files, no set, eeg. or other format?

**Comment by umair-hassan** [comment]
Hi Ola,

Thanks for your message.

1. Can you attach the mat file you are trying to open, I will be able to have a look over it.
2. BEST only works with mat files at the moment, can you also provide details of the use case to employ the .set or .eeg or other binary formats, then we could try to add those features in the future releases if applicable.
